public/stylesheets/main.css

```diff
@@ -1,108 +1,59 @@
-html,
-body {
-    padding: 0;
-    margin: 0;
-    box-sizing: border-box;
-}
-body {
+body{
     font-size: 16px;
-    line-height: 1.5;
-    background: #f2f2f2;
-}
-.flashMessageContainer {
-    position: fixed;
-    top: 90px;
-    left: 10%;
-    width: 80%;
-    height: 2.5rem;
-    max-width: 100%;
-    background-color: transparent;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-}
-.flashMessageContainer .message {
-    flex-basis: 100%;
-}
-section {
-    margin-top: 66px;
-    padding: 4rem 0;
-}
-.hero {
+    margin: 0;
     padding: 0;
-    margin-top: 0;
-    background-image: url('/images/hero.jpg');
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
-    height: 60vh;
-}
-.hero .row {
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    align-items: center;
-    height: 60vh;
-}
-.hero .contentContainer {
-    display: flex;
-    flex-direction: column;
-    justify-content: space-between;
-    align-items: center;
-    height: 30vh;
-}
-.hero h1,
-.hero p,
-.hero a {
-    text-align: center;
-    color: white;
+    line-height: 1.5;
+    background:#f2f2f2 ;
 }
-ul {
+ul{
     padding: 0;
     margin: 0;
     list-style: none;
 }
-ul > li {
+ul>li{
     padding: 5px;
 }
-#Open {
+#open{
```

Comment:
please don't select in large scale apps as this tags by ul li for custom styles. only if you spacify global styles to them like this.

don't use id selectors all over.